java/src/main/java/org/eclipse/ditto/client/internal/bus/AdaptableBus.java

```diff
@@ -109,6 +109,13 @@ SubscriptionId subscribeForAdaptableWithTimeout(Classification tag,
      */
     boolean unsubscribe(@Nullable SubscriptionId subscriptionId);
 
+    /**
+     * Closes the executor of the adaptable bus .
+     *
+     * @since 1.2.2
```

Comment:
the `@since` can be removed as this is in the `internal` package.
`@since` should also only be done for major versions ( e.g. `@since 1.3.0`)